src/QMCApp/qmcapp.cpp

```diff
@@ -220,7 +220,7 @@ int main(int argc, char** argv)
     timingDoc.newDoc("resources");
     output_hardware_info(qmcComm, timingDoc, timingDoc.getRoot());
     timer_manager.output_timing(qmcComm, timingDoc, timingDoc.getRoot());
-    qmc->ptclPool->output_particleset_info(timingDoc, timingDoc.getRoot());
+    qmc->getParticlePool().output_particleset_info(timingDoc, timingDoc.getRoot());
```

Comment:
This line reminds me the blurry boundary betwen qmcapp and QMCMain.
We will need to follow the following direction as we clean up old code.
qmcpapp is a layer that handles ensemble job.
All the timer manager, info.xml writing should be moved inside QMCMain.

Reply:
Thanks, that helps!